### ISSUE TITLE
Is there a way to see the hyperparameters Fasttext uses for the model when using autotune-validation?

### ISSUE THREAD

**Issue by Umar-Ayub** [facebookresearch/fastText#961]
Hi I am trying to determine what are the hyperparameters of a trained model I have. I used autotune validation to do my training like the following: 

`model = fasttext.train_supervised(train_file, autotuneValidationFile=val_file, autotuneDuration=14400, verbose=2)`

really want to see what hyperparameters it used and can't find anything on this in the docs or online. Any idea how I could find these things out such as the loss function it used? the minn the maxn. Thanks

**Comment by Celebio** [comment]
Hi @Umar-Ayub ,
In order to see the parameters the autotune is trying, you can have a look on [the discussion on this issue](https://github.com/facebookresearch/fastText/issues/887).

The loss function it uses is `softmax` unless you specify another one manually.
Best regards,
Onur
